src/ci/docker/host-x86_64/dist-various-2/build-wasi-toolchain.sh

```diff
@@ -2,15 +2,15 @@
 
 set -ex
 
-# Originally from https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/clang+llvm-11.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz
-curl https://ci-mirrors.rust-lang.org/rustc/2021-01-14-clang%2Bllvm-11.0.1-x86_64-linux-gnu-ubuntu-16.04.tar.xz | \
+# Originally from https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.0/clang+llvm-14.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz
+curl https://ci-mirrors.rust-lang.org/rustc/2022-05-10-clang%2Bllvm-14.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz | \
```

Comment:
It looks like dist-various-2 is using 20.04 Ubuntu now -- should we be downloading a 20.04 image here as well?

Reply:
Only an ubuntu 18.04 tarball is provided by https://github.com/llvm/llvm-project/releases/tag/llvmorg-14.0.0 It should be fine though due to glibc being mostly backwards compatible.